packages/nuxt/src/components/plugins/loader.ts

```diff
@@ -90,43 +90,57 @@ export const LoaderPlugin = (options: LoaderOptions) => createUnplugin(() => {
               switch (modifier) {
                 case 'Visible':
                 case 'visible-':
-                  imports.add(genImport(options.clientDelayedComponentRuntime, [{ name: 'createLazyVisibleComponent' }]))
+                  if (!code.includes('createLazyVisibleComponent')) {
```

Comment:
do we need this? (or, maybe we could run `findImports` instead of matching across the whole code)?

Reply:
This is to prevent potential conflicts when users mix both auto-imports and explicit imports within a single SFC. I’ll take a closer look and evaluate whether `findImports` or the scope tracker would be a better fit for this case.

Reply:
`createLazyVisibleComponent` is not exposed to end users - it's a purely internal utility